runtime/oti/j9nonbuilder.h

```diff
@@ -1579,9 +1579,10 @@ typedef struct J9ExceptionHandler {
 	U_32 exceptionClassIndex;
 } J9ExceptionHandler;
 
-#if defined(__xlC__) || defined(J9ZOS390)  /* Covers: Z/OS, AIX, Linux PPC*/
+//JACKIE: not sure where defined(__open_xl__) condition should go
+#if defined(__xlC__) || defined(J9ZOS390) || defined(__open_xl__)  /* Covers: Z/OS, AIX, Linux PPC*/
```

Comment:
Please finish each sentence with a period, spell z/OS correctly and sort platforms:
```c
#if defined(__xlC__) || defined(J9ZOS390) || defined(__open_xl__) /* Covers: AIX, Linux PPC and z/OS. */
```